docs/reference/configuration.rst

```diff
@@ -955,16 +955,16 @@ CMS_TOOLBAR_SIMPLE_STRUCTURE_MODE
 =================================
 
 default:
-    ``False``
+    ``True``
 
 Enable or disable the absolute positioning within the structure board.
 
-By default the absolute positioned board is activated. Setting this attribute
-to ``False`` will prevent the absolute positioning and order the placeholders
-beneath each other. Static placeholders have a visual gap in the simple mode.
+By default the absolute positioned board is deactivated. Setting this attribute
+to ``False`` will allow the absolute positioning used in versions prior to 3.2.
```

Comment:
This isn't very clear at all. Consider:

"By default, starting in django CMS 3.2, the structure board will now ignore absolutely positioned elements. To use a structure board which respects absolutely positioned elements, set this flag to False. NOTE: This option will be removed in version 3.3."